app/controllers/tournaments_controller.rb

```diff
@@ -20,6 +40,13 @@ def scores
   end
 
   def success
+    begin
+      results = Result.select("results.answers").where(tournament_id: params[:id])
+      statistics = successPerQuestion(results)
```

Comment:
use snake_case
You can use rubocop to find the violations

Reply:
Why should it be snake_case?

Reply:
The ruby convention is that functions and variables are snake_case, classes and modules are CamelCase